src/main/scala/Block.scala

```diff
@@ -158,7 +158,8 @@ case class Block(center: (Double,Double,Double), faces: Seq[Face]) {
             } else None
           }
         }
-      } map (_.distinct) map (_.reverse) // Reverse is needed to keep the same ordering as faces list since elements are added to the front
+        // Reverse is needed to keep the same ordering as faces list since elements are added to the front
```

Comment:
It looks like the end of this comment was down on line 163. I think the comment here makes sense as is, so maybe you could delete that part?